src/script/components/calling/FullscreenVideoCall.tsx

```diff
@@ -152,11 +151,11 @@ const FullscreenVideoCall: React.FC<FullscreenVideoCallProps> = ({
   const showSwitchMicrophone = availableMicrophones.length > 1;
 
   const [hasUnreadMessages, setHasUnreadMessages] = useState(false);
-  const updateUnreadCount = (unreadCount: number) => setHasUnreadMessages(unreadCount > 0);
+  const {unreadMessagesCount} = useAppState();
+
   useEffect(() => {
-    amplify.subscribe(WebAppEvents.LIFECYCLE.UNREAD_COUNT, updateUnreadCount);
-    return () => amplify.unsubscribe(WebAppEvents.LIFECYCLE.UNREAD_COUNT, updateUnreadCount);
-  }, []);
+    setHasUnreadMessages(unreadMessagesCount > 0);
+  }, [unreadMessagesCount]);
```

Comment:
I'm not sure we need `hasUnreadMessages` to be separate local state (`useState` + `useEffect`). What about just creating a variable inside component `const hasUnreadMessages  = unreadMessagesCount > 0`, and always relaying on global zustand state (we will get rid of 1 extra re-render)? 😄